src/components/common/Filter/Filter.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import FilterComponent from './FilterComponent';
+import FilterButton from './FilterButton';
+
+const Filter = props => (
+  <FilterButton
+    activePage={props.activePage}
+    limit={props.limit}
+    selected={props.selected}
+    filterAction={props.filterAction}
+    disabled={props.isLoading}
+    option={props.filterData[1]}
+  >
+    <React.Fragment>
```

Comment:
Please delete `<React.Fragment>`. This component should be:
```jsx
const Filter = props => (
  <FilterButton
    activePage={props.activePage}
    ...
    option={props.filterData[1]}
  >
    {props.filterData.map((data, index) => (...)}
  </FilterButton>
);
```
Or is React complaining when you don't use `React.Fragment`